prow/config_knative.yaml

```diff
@@ -1063,6 +1063,8 @@ periodics:
     release: "1.0"
   - branch-ci: true
     release: "1.1"
+  - branch-ci: true
```

Comment:
Please also add the dot-release job for release 1.2 - above L103.

Reply:
Done